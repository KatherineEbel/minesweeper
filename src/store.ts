import { configureStore } from '@reduxjs/toolkit'
import {reducer as minesweeper} from 'modules/MinesweeperRedux'

export const store = configureStore({
  reducer: {
    minesweeper
  }
})

export type RootState = ReturnType<typeof store.getState>
export type AppDispatch = typeof store.dispatch