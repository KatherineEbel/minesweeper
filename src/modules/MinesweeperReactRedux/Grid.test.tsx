import {render} from '@testing-library/react'
import GridRedux from 'modules/MinesweeperReactRedux/Grid'
import React from 'react'
import {Provider} from 'react-redux'
import {store} from 'store'

test('Grid renders correctly', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <GridRedux/>
    </Provider>
  )
  expect(asFragment()).toMatchSnapshot()
})