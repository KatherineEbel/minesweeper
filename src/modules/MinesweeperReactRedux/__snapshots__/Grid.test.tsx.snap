// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid renders correctly 1`] = `
<DocumentFragment>
  <div
    class="mine-field css-52ne6"
    size="9"
  >
    <div
      class="cell-0-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-0-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-1-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-2-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-3-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-4-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-5-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-6-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-7-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-0 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-1 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-2 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-3 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-4 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-5 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-6 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-7 css-hgqh5v"
      role="cell"
    >
      10
    </div>
    <div
      class="cell-8-8 css-hgqh5v"
      role="cell"
    >
      10
    </div>
  </div>
</DocumentFragment>
`;
